osu.Game.Rulesets.Osu/Difficulty/Evaluators/AimEvaluator.cs

```diff
@@ -15,6 +15,11 @@ public static class AimEvaluator
         private const double slider_multiplier = 1.35;
         private const double velocity_change_multiplier = 0.75;
 
+        private static double CosSigmoid(double value, double range = 1)
```

Comment:
This should go to the bottom of the file, or maybe even somewhere common (framework? some common math class here? not sure, client maintainers please help)

Reply:
99% sure that it doesn't exist in frameworks
i even didn't find it's own wikipedia page